seedlet-blocks/block-template-parts/header.html

```diff
@@ -8,14 +8,12 @@
 
 <!-- wp:site-title {"align":"center"} /-->
 
-<!-- wp:paragraph {"align":"center","fontSize":"small"} -->
-<p class="has-text-align-center has-small-font-size">is a curious botanist</p>
-<!-- /wp:paragraph -->
+<!-- wp:site-tagline {"align":"center","fontSize":"small"} /-->
```

Comment:
I think this needs to be updated to `textAlign` after @Addison-Stavlo's latest changes.
```suggestion
<!-- wp:site-tagline {"textAlign":"center","fontSize":"small"} /-->
```